### DISCUSSION TITLE
Real fullscreen?

### DISCUSSION THREAD

**Discussion by shadow82** [photoprism/photoprism#3333] [General]
Hi!
I deployed PhotoPrism and I think it'll stay with me for long time on my homeserver.
Tell me one thing: When I start view pictures in fullscreen, it is not a real fullscreen (same behavior for slideshows_

![image](https://user-images.githubusercontent.com/16290219/230054834-9d26f6da-7970-449a-839d-2b4abd24016e.png)

How to make it real full fullscreen?

**Comment by graciousgrey** [comment]
We need to build a new photo viewer for this: https://github.com/photoprism/photoprism/issues/1307

**Reply by shadow82** [reply]
Understood. What's the ETA for new photo viewer module?

**Reply by graciousgrey** [reply]
:point_right: https://docs.photoprism.app/user-guide/faq/ Question 2 ;)
